lib/Controller/SignalingController.php

```diff
@@ -253,6 +259,16 @@ private function validateBackendRequest($data) {
 		return hash_equals($hash, strtolower($checksum));
 	}
 
+	/**
+	 * Return the body of the backend request. This can be overridden in
+	 * tests.
+	 *
+	 * @return string
+	 */
+	protected function getInputStream() {
+		return file_get_contents('php://input');
```

Comment:
`IRequest::getContent` should do the same

Reply:
Doesn't `IRequest::getContent` already parse the contents if JSON is sent? I need the raw body to verify the checksum.